rotkehlchen/tests/api/test_bitcoin.py

```diff
@@ -62,7 +68,7 @@
 TEST_BITCOIN_XPUB_2 = 'xpub68V4ZQQ62mea7ZUKn2urQu47Bdn2Wr7SxrBxBDDwE3kjytj361YBGSKDT4WoBrE5htrSB8eAMe59NPnKrcAbiv2veN5GQUmfdjRddD1Hxrk'  # noqa: E501
 
 
-def _check_xpub_addition_outcome(outcome, xpub):
+def _check_xpub_addition_outcome(outcome: Any, xpub: str) -> None:
```

Comment:
Again same here. Don't use `Any`. What is `result` I see `assert_proper_sync_response_with_result` also uses it, which is wrong. It should be most probably a dict right? Or can it also be a list? In which case you need a union.

If you need both then you would need to create different `assert_` functions which would check the result internally and return it. Since this is tests should be fine